tests/jballerina-unit-test/src/test/resources/test-src/types/string/string-value-test.bal

```diff
@@ -20,3 +20,9 @@ function testError() returns int {
     error err = error(smiley);
     return err.reason().length();
 }
+
+function testArrayStore() returns string[] {
```

Comment:
Shall we also add test for a multi element array also, just to be on the safe side?